bin/img.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# are there images to optimize?
+fileschanged=$(git diff --diff-filter=ACM --name-only HEAD | grep ".png\|.jpg\|.svg" | wc -l)
+if [ $fileschanged == 0 ]
+then
+    echo "No images to optimize. Did you remember to stage them?"
+    exit 1
+fi
+
+msgs=()
+
+# ask to optimize with tinypng, we don't want to do this multiple times
+read -p "Tinypng png and jpg images? " -n 1 -r
+echo    # new line
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+    # is there a tinyPNG API key
+    key="${HOME}/.tinypng"
+    if ! [ -f "$key" ]; then
+        echo "TinyPng API key not found."
+        # let's save one, shall we?
+        echo "What is your API key? (You can sign up for one here: https://tinypng.com/developers)"
+        read apikey
+        printf $apikey > ${HOME}/.tinypng
+        echo "API key saved in $key"
+    fi
+
+    # tinypng png and jpg images
+    git diff --diff-filter=ACM --name-only HEAD | grep ".png\|.jpg" | xargs tinypng
```

Comment:
Pointing this to `xargs ./node_modules/tinypng-cli/tinypng-cli.js` worked for me.